test-app/tests/integration/components/input-field-test.gts

```diff
@@ -110,6 +110,26 @@ module('Integration | Component | InputField', function (hooks) {
     );
   });
 
+  test('it renders a hint and label block', async function (assert) {
+    await render(<template>
+      <InputField
+        @label="Label"
+        type="text"
+        @hint="Hint text visible here"
+        data-input
+      >
+      <:label>Extra label content</:label>
+      <:hint>Extra hint content</:hint>
+      </InputField>
+    </template>);
+
+    const label = '[data-label]';
+    const hint = '[data-hint]';
+
+    assert.dom(label).hasText('Label Extra label content'); 
+    assert.dom(hint).hasText('Hint text visible here Extra hint content'); 
```

Comment:
nit: Instead of doing a `find` can these be the following to match the other tests above?

```suggestion
    assert.dom('[data-hint]').hasText('Hint text Extra hint content');
    assert.dom('[data-label]').hasText('Label Extra label content');
```